autoPyTorch/api/base_task.py

```diff
@@ -1223,10 +1351,29 @@ def refit(
 
         return self
 
-    def fit(self,
-            dataset: BaseDataset,
-            pipeline_config: Optional[Configuration] = None,
-            split_id: int = 0) -> BasePipeline:
+    def fit_pipeline(
+        self,
+        configuration: Configuration,
```

Comment:
Since we can take both `xxx_train` and `dataset`, it is better to force users to pass variables by keywords.
To achieve it, we need the `*` before the beginning of the keywords.
```suggestion
    def fit_pipeline(
        self,
        configuration: Configuration,
        *,
```

Reply:
Okay, I have implemented **dataset_kwargs

Reply:
No no, you can just add * as in the suggestion and that works nicer than kwargs which hides information

Reply:
oh okay, I got it. Thanks, I actually did not know about this functionality.